src/navigation/nav_bringup/config/nav2_params.yaml

```diff
@@ -4,6 +4,8 @@
     global_frame: track
     local_frame: odom
     robot_base_frame: base_footprint
+    # odom_topic: odometry/filtered  # this is actually quite noisy and often drops to 0 m/s for some reason
```

Comment:
Fixes on the post-ebs test review branch solve the root cause of this. Note to go back to the filtered topic once those changes are validated. IMU odom will be oriented wrt. earth's magnetic north, so velocities may be a bit weird. While filtered odom is zeroed based on initial heading through RL.